src/library/dynamo.js

```diff
@@ -128,3 +128,5 @@ export const batchPutIntoDynamoDb = async (AWS, recs, tName,
     throw err;
   }
 };
+
+// CR: Mickey: add singlePutIntoDynamoDb method?
```

Comment:
@MickeyMcc I did not see a great performance difference between bulk and single put when there is only one item to be inserted/ updated. Therefore, I did not add the singlePut method. Please let me know if you think otherwise.